lib/pavilion/plugins/results/table.py

```diff
@@ -28,21 +28,28 @@ def get_config_items(self):
             ),
             yc.StrElem(
                 'has_header', default='False',
-                help_text="Set True if there is a column for row names."
+                help_text="Set True if there is a column for row names. Will "
+                          "create dictionary of dictionaries."
             ),
             yc.ListElem(
                 'col_names', required=False, sub_elem=yc.StrElem(),
                 help_text="Column names if the user knows what they are."
+            ),
```

Comment:
Would you be ok with making this the default (and thus probably renaming it 'by_column' or something instead). It breaks backwards compatability, but that's ok.